tests/analysis/adaptation/test_adaptation.py

```diff
@@ -132,5 +146,30 @@ def test_calculate_bc_ratio_returns_gdf(
             mocked_adaptation.adaptation_collection.adaptation_options
         ):
             assert _result[f"{_option.id}_bc_ratio"].sum(axis=0) == pytest.approx(
-                _nof_rows * (4.0 + i) / (1.0 + i)
+                len(_result.index) * (4.0 + i) / (1.0 + i)
             )
+
+    def test_output_gdf_can_be_exported_to_gpkg(
+        self,
+        valid_adaptation_config_with_input: tuple[
+            AnalysisInputWrapper, AnalysisConfigWrapper
+        ],
+        test_result_param_case: Path,
+    ):
+        # 1. Define test data.
+        _adaptation = Adaptation(
+            valid_adaptation_config_with_input[0], valid_adaptation_config_with_input[1]
+        )
+
+        # 2. Run test.
+        _result = _adaptation.execute().results_collection[0]
```

Comment:
Check:
1. whether `_result` is of the expected type,
2. `.results_collection` of the expected length, 
3. `_results_collection[0]` of the expected type, 
4. then you can move on.

```suggestion
        _result_wrapper = _adaptation.execute()
        assert is instance(_result_wrapper, AnalysisResultWrapperProtocol)
        assert _result_wrapper.is_valid() # Technically this already validates 2. and 3.
        _result = _result_wrapper.results_collection[0]
```

Reply:
Adjusted